tests/spec/fs.stat.spec.js

```diff
@@ -91,4 +91,32 @@ describe('fs.stat', function() {
       });
     });
   });
+
+  it('(promise) should be a function', function() {
+    var fs = util.fs();
+    expect(fs.promises.stat).to.be.a('function');
+  });
+  
+  it('should return a promise', function() {
+    var fs = util.fs();
+    expect(fs.promises.stat()).to.be.a('Promise');
+  });
+  
+  it('(promise) should return a stat object if file exists', function(){
+    var fs = util.fs();
+
+    return fs.promises.stat('/')
```

Comment:
Style nit: drop your function calls down to all have the same indent (e.g. `stat`, `then`), to set them out visually.

```js
return fs.promises
  .stat('/')
  .then(result => { ...
```